invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -145,12 +145,17 @@ export const RDMRecordSearchBarElement = withState(
     currentQueryState,
   }) => {
     const placeholder = passedPlaceholder || i18next.t("Search");
+
+    const onSearch = () => {
+      updateQueryState({ ...currentQueryState, queryString });
+    };
+
     const onBtnSearchClick = () => {
-      updateQueryState({ ...currentQueryState, filters: [], queryString });
+      onSearch();
```

Comment:
I am not sure why we were resetting the filters on search but this change will affect several searchbars (like the upload and communities searchbars) as it's `RDMRecordSearchBarElement`, just want to double check that you tested all of them :)

Reply:
i think a misunderstanding from the earlier days of development, also don't think there were any filters before. Thanks for the reminder, all tested :)